tests/core/test_hook_methods.py

```diff
@@ -60,40 +58,13 @@ def legend_entry_data(pyramid_oereb_test_config, dbsession, transact, file_adapt
     yield legend_entries
 
 
-def test_get_symbol_invalid_theme_code(pyramid_oereb_test_config):
```

Comment:
is there no use for these removed tests ?

Reply:
yes there is but not on Core

Reply:
Yes there is none-since we do not have any implementation in the core. But only the hook-method dummy.

Reply:
OK. we shoudl just not forget to add them later on in a convenient place.